app/src/main/java/org/simple/clinic/enterotp/EnterOtpScreen.kt

```diff
@@ -137,6 +144,24 @@ class EnterOtpScreen : BaseScreen<
       layoutInflater: LayoutInflater,
       container: ViewGroup?) = ScreenEnterotpBinding.inflate(layoutInflater, container, false)
 
+  override fun events() =  Observable
+      .mergeArray(
+          otpSubmits(),
+          resendSmsClicks()
```

Comment:
Also can we reformat all the files whenever we commit something please?

Reply:
This is done in the later commit when we remove all the unused code. PTAL?